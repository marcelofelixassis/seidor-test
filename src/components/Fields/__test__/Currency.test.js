import React from 'react';
import { shallow } from 'enzyme';
import Currency from '../currency';

describe('Test currency input', () => {
    it('basic snapshot render', () => {
        const component = shallow(<Currency  name={"teste"} value={"123"} onChange={jest.fn()}/>);
        expect(component).toMatchSnapshot();
    })
});