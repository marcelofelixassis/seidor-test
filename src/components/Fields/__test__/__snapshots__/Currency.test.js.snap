// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test currency input basic snapshot render 1`] = `
<NumberFormat
  allowEmptyFormatting={false}
  allowLeadingZeros={false}
  allowNegative={true}
  decimalScale={2}
  decimalSeparator=","
  displayType="input"
  fixedDecimalScale={true}
  inputMode={
    <input
      name="teste"
    />
  }
  isAllowed={[Function]}
  isNumericString={false}
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onMouseUp={[Function]}
  onValueChange={[Function]}
  placeholder="R$ 0,00"
  prefix="R$ "
  suffix=""
  thousandSeparator="."
  thousandsGroupStyle="thousand"
  type="text"
  value="123"
/>
`;

exports[`renders the component 1`] = `
<NumberFormat
  allowEmptyFormatting={false}
  allowLeadingZeros={false}
  allowNegative={true}
  decimalScale={2}
  decimalSeparator=","
  displayType="input"
  fixedDecimalScale={true}
  inputMode={
    <input
      name="teste"
    />
  }
  isAllowed={[Function]}
  isNumericString={false}
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onMouseUp={[Function]}
  onValueChange={[Function]}
  placeholder="R$ 0,00"
  prefix="R$ "
  suffix=""
  thousandSeparator="."
  thousandsGroupStyle="thousand"
  type="text"
  value="123"
/>
`;
